ducktape/command_line/parse_args.py

```diff
@@ -77,6 +77,10 @@ def create_ducktape_parser():
     parser.add_argument("--test-runner-timeout", action="store", type=int, default=1800000,
                         help="Amount of time in milliseconds between test communicating between the test runner"
                              " before a timeout error occurs. Default is 30 minutes")
+    parser.add_argument("--ssh-checker-function", action="store", type=str, nargs="+",
```

Comment:
Nit: Can this flag be specified multiple times or do you specify multiple functions after a single flag? It's not clear from the description

Reply:
cool, as long as it is clear from the docs. I was confused because actual checkers on the cluster object are a list